site/app/libraries/database/DatabaseQueries.php

```diff
@@ -7344,6 +7346,18 @@ public function updateGradeable(\app\models\gradeable\Gradeable $gradeable) {
                     WHERE g_id=?",
                     $params
                 );
+
+                // Reset notification state of gradeable if scores need to be released again
```

Comment:
Do we necessarily want to re-notify every time any part of the gradable has been updated? If so do we want to warn the instructor before they submit the update?